Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -66,6 +69,85 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         },
 
         /**
+         * Sets fields of edited version.
+         *
+         * @method _setVersionFields
+         * @private
+         * @param {Function} callback
+         */
+        _setVersionFields: function (callback) {
+            var service = this,
+                content = this.get('content'),
+                resources = content.get('resources'),
+                contentType = resources.ContentType,
+                version = this.get('version'),
+                fields = content.get('fields'),
+                languageCode = this.get('languageCode'),
+                baseLanguageCode = this.get('baseLanguageCode'),
+                languageCodeForLoadContent = baseLanguageCode ? baseLanguageCode : languageCode,
+                setDefaultFields = false,
+                languageInTranslationsList;
+
+            languageInTranslationsList = Y.Array.find(
+                                            content.get('currentVersion').getTranslationsList(), 
```

Comment:
btw CS issue and instead if ( condition ) { return true } you can directly return the result of the condition, something like:
```js
languageInTranslationsList = Y.Array.find(
    content.get('currentVersion').getTranslationsList(), function (translation) {
        return (translation === languageCodeForLoadContent);
    }
);
```